### PR TITLE
 add "join table" for visits

### PR DESCRIPTION
**Pull request by kal2944**
### **IN THIS PROJECT I HAVE UPDATED THE FOLLOWING:**

- [x] Created a table named vets and its columns
- [x]  Create a "join table" called specializations to handle the relationship. For the many-to-many relationship between the  animals and vets tables
- [x] Inserted data for vets
- [x] Inserted data for specialization
- [x] Inserted data for visits
- [x] Wrote queries to answer the questions

SCREENSHOTS OF THE QUERIES

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -57,3 +57,38 @@ UPDATE animals set owner_id = 4 WHERE name = 'Charmander' or name = 'Squirtle' o
 UPDATE animals set owner_id = 5 WHERE name = 'Angemon' or name = 'Boarmon';
 
 UPDATE owners SET full_name = 'Jennifer Orwell' WHERE id = 2;
+
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('William Tatcher', 45, '2000-04-23'),('Stephanie Mendez', 64, '1981-05-04'),('Jack Harkness', 38, '2008-08-08');
+
```
Comment (TuyishimireEric) [inline data.sql]: - [ ] **Good job so far 👏** however, there is a missing input due to the given  data *Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019* is missing in your database, Kindly fix this to get the expected output 🙏

**Review by TuyishimireEric — APPROVED** [review]
# Status: REQUIRED CHANGES :recycle: 
### @kal2944 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight 
✔ Used correct SQL commands👍 
✔ No Linter Errors👍 


## Required Changes ♻️

- []  Check the comments below 

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (kal2944) [reply data.sql]: Thank you, I have added the missing data and replaced the screenshot with the new one in the pull request.

**Review by muneebulrehman — APPROVED** [review]
Hi @kal2944 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- Descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**